raiden/ui/cli.py

```diff
@@ -292,11 +306,15 @@ def app(address,
         blockchain_service.discovery(discovery_contract_address)
     )
 
-    # default database directory
-    raiden_directory = os.path.join(os.path.expanduser('~'), '.raiden')
+    if database_dir is None:
+        # default database directory
+        raiden_directory = os.path.join(os.path.expanduser('~'), '.raiden')
+    else:
+        raiden_directory = database_dir
+
     if not os.path.exists(raiden_directory):
         os.makedirs(raiden_directory)
-    database_path = os.path.join(raiden_directory, 'log.db')
```

Comment:
okay, I did that:
WAL should now be in `~/.raiden/pex(address)/log.db` and snapshots in `~/.raiden/pex(address)/snapshots/data.pickle`